Base/usr/share/man/man1/RelationalStudio.md

```diff
@@ -0,0 +1,32 @@
+## Name
+
+![Icon](/res/icons/16x16/app-relational-studio.png) Relational Studio - SerenityOS SQL Manager
+
+[Open](file:///bin/RelationalStudio)
+
+## Synopsis
+
+```**sh
+$ RelationalStudio [script.sql | database.db]
+```
+
+## Description
+
+RelationalStudio is a graphical SQL manager program that allows the user to create and edit
+SQL scripts.
+
+## Arguments
+
+* `script.sql`: SQL script to open, edit or run
+* `database.db`: SQL database to open and run scripts against
+
+## Examples
+
+```sh
+$ RelationalStudio
+$ RelationalStudio /home/anon/Documents/sql/insert_values.sql
+```
+
+## Caveats
+
+Connecting to a database has not been implemented.
```

Comment:
That's not necessary in a manpage. Let's just have the user discover this for themselves; I'm sure it will be implemented to some degree soon enough!